api/camunda/camunda-openapi.yaml

```diff
@@ -1889,9 +1889,9 @@ components:
         endDate:
           type: string
           format: date-time
-        bpmnProcessId:
```

Comment:
@nicpuppa - should this be here?

Reply:
Looks like has been updated here: https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L2414